### PR TITLE
api: remove `.json` in the game manifest route

### PR DESCRIPTION
**Pull request by dag7dev**

### CODE REVIEW

**Review by avivace — APPROVED** [review]

File: hhub/views.py [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+from django.http import HttpResponse, JsonResponse, FileResponse
 
 from hhub.models import Entry
 from hhub.serializers import EntrySerializer
 import json
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 
+# given a filename, returns that file
+# /entry/<entry-slug>/<filename>
+def entry_download_file(request, pk, filename):
```
Comment (avivace) [inline hhub/views.py:10]: Why are you replacing the route serving the game manifest with one serving every file?

**Review by avivace — CHANGES_REQUESTED** [review]

**Comment by avivace** [comment]
Hey, I appreciate your effort, but this PR is quite confusing for me:


1. We will not use Django to serve files or any other assets related to entries. Therefore, it makes no sense to expose a route that has to:
    - check if a file is actually there
    - handle the game.json in another URL
    - send the file through Django
2. At some point, we may decide to enrich the Django model to accomodate more values from the game.json . In that case, we may want to serialize the Entry instead of sending the JSON from disk. Your change doesn't enable this as you are trying to merge the "serve_files" route with the one that should serve the game manifest
3. The game attachments are already organized in the FS and accessible simply with the names exposed in the game manifest. Django doesn't need to serve those because it's enough to clone the database repository and tell Nginx (or whatever front web server) to serve content of the "database/entries/" folder under "entries". There's no need for an actual API route there..

File: hhub/views.py (hunk 1 above)
Comment (dag7dev) [reply hhub/views.py:10]: Not sure if I got it;
What I tried to achieve, in poor words is: "hey, if I ask for file named 2048.gb, gimme that file if exists in <entry-slug> and it is called <filename>"

**Comment by dag7dev** [comment]
@avivace from README
> GET /entry/<entry-slug>/<filename>
I thought that route wasn't implemented yet, therefore I thought it was missing :)

**Pull request closed without merging**